packages/gatsby-image/src/index.js

```diff
@@ -250,24 +258,30 @@ class Image extends React.Component {
       itemProp,
     } = convertProps(this.props)
 
+    const shouldDisplay = this.state.imgLoaded || this.state.fadeIn === false
+    const shouldFadeIn = this.state.fadeIn === true && !this.state.imgCached
+    const durationFadeIn = `0.5s`
+
+    const imageStyle = {
+      opacity: shouldDisplay ? 1 : 0,
+      transition: shouldFadeIn ? `opacity ${durationFadeIn}` : `none`,
+      ...imgStyle,
+    }
+
     const bgColor =
       typeof backgroundColor === `boolean` ? `lightgray` : backgroundColor
 
-    const initialDelay = `0.25s`
+    const delayHide = {
```

Comment:
I would go for you suggestion
```suggestion
    const delayHideStyle = {
```

Reply:
Sure!